### ISSUE TITLE
Document the following banner behavior on IOS/IOS-XE

### ISSUE THREAD

**Issue by ktbyers** [napalm-automation/napalm-ios#23]
I did some tests on this. 

It looks like the banner requires the EXT character (ASCII 3). This looks like a cntl-C in the file, but as a single character.

```
banner motd ^C
my banner test
^C
```

I wrote it as follows:

```
>>> ext_char = chr(3)
>>> with open("my_config.conf", "a") as f:
...   f.write("banner motd {}\n".format(ext_char))
...   f.write("my banner test\n")
...   f.write("{}\n".format(ext_char))
... 
>>> quit()
```

This appended an existing file and then I moved it to where I wanted in the file.

kcelenza [8:50 PM]  

```
in vim insert, you can also type `<ctrl>+ V` release only the V, then type `C`
```

**Comment by ktbyers** [comment]
In this PR:

https://github.com/napalm-automation/napalm/pull/379/files

**Comment by dbarrosop** [comment]
Fixed via napalm-automation/napalm#379
